### PR TITLE
Add: navbar sign in button

### LINKED ISSUE #54: Add Login and Sign Up Buttons
Screenshot 2021-07-09 at 00 32 13

### PR DESCRIPTION
**Pull request by LayanJay**

### CODE REVIEW

File: pages/event.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+          <HappeningNow posts={posts} picture={picture} />
+          <UpcomingSection posts={posts} picture={picture} />
+        </Container>
+      </section>
+      <Footer />
+    </Layout>
+  )
+}
+
+export default event
```
Comment (sithumonline) [inline pages/event.tsx]: I feel export is not okay. 
cc @Shehanka

File: pages/event/[id].tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        className="bg-no-repeat bg-right-top bg-cover py-10"
+      >
+        <Container>{title}</Container>
+      </section>
+      <Footer />
+    </Layout>
+  )
+}
+
+export default SingleEventPage
```
Comment (sithumonline) [inline pages/event/[id].tsx]: File naming is wrong. 
cc @Shehanka

**Review by sithumonline — CHANGES_REQUESTED** [review]

**Comment by LayanJay** [comment]
CI is failing because of a type error that is not in my scope. (I didn't even touch those files) cc @sithumonline

**Comment by LayanJay** [comment]
Why do we use material UI components when we use tailwind?? cc @sithumonline

**Comment by sithumonline** [comment]
> CI is failing because of a type error that is not in my scope. (I didn't even touch those files) cc @sithumonline

It looks like issue with `react-hook-form`

**Comment by sithumonline** [comment]
> Why do we use material UI components when we use tailwind?? cc @sithumonline

I told @Akalanka47000 to do Login and Signup so I think Akalaka cp n pt example what is gave to him. 
https://github.com/sliit-foss/rsvp-frontend/pull/57/commits/7cc5b4405eb44b8ebaeea5d2c43f5e134d3c7244

File: pages/event/[id].tsx (hunk 2 above)
Comment (sithumonline) [reply pages/event/[id].tsx]: It okay.. I just got this 
https://nextjs.org/docs/basic-features/pages#pages-with-dynamic-routes

**Review by sithumonline — APPROVED** [review]

**Pull request merged**